source/manual/rules-for-getting-production-access.html.md

```diff
@@ -11,17 +11,53 @@ These rules apply to developers, SREs, and technical architects in the GOV.UK pr
+
+- is aware of our processes and standards around [code review](https://gds-way.cloudapps.digital/manuals/code-review-guidelines.html)
+- understands the responsibilities that [releasing code](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#deployment) brings with it
+- knows how to roll back to an older release if there are any issues
+- knows [how to get help](https://docs.publishing.service.gov.uk/manual/ask-for-help.html) from someone with more access if they need it
+
+### Production Admin access
+
+- Permission to read & write [production](https://github.com/alphagov/govuk-secrets/blob/main/puppet_aws/hieradata/production_credentials.yaml) and [staging](https://github.com/alphagov/govuk-secrets/blob/main/puppet_aws/hieradata/staging_credentials.yaml) [hieradata](https://docs.publishing.service.gov.uk/manual/encrypted-hiera-data.html#what-to-do-when-someone-gets-production-access) in govuk-secrets using GPG
+- Permission to read & write from the [Pass](https://github.com/alphagov/govuk-secrets/tree/main/pass) in govuk-secrets store using [GPG](https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/.gpg-id)
+- Access to [Production Deploy Jenkins](https://deploy.blue.production.govuk.digital/) and [Staging Deploy Jenkins](https://deploy.blue.staging.govuk.digital/) to deploy applications via the [GOV.UK Production GitHub team](https://github.com/orgs/alphagov/teams/gov-uk-production)
+- [SSH access](https://docs.publishing.service.gov.uk/manual/howto-ssh-to-machines.html) to production and staging servers via [govuk-puppet](https://github.com/alphagov/govuk-puppet)
 - AWS [PowerUser Access](https://github.com/alphagov/govuk-aws-data/blob/master/data/infra-security/production/common.tfvars) via the `role_poweruser_user_arns` role
 - [Google Cloud Platform (GCP)](/manual/google-cloud-platform-gcp.html) access to role to manage [static mirrors](/manual/fall-back-to-mirror.html) and DNS
 - Signon "Super Admin" access in production
-- GOV.UK PaaS [Space developer](https://docs.cloud.service.gov.uk/orgs_spaces_users.html#space-developer)
-  access to all spaces in [the govuk_development organisation](https://admin.cloud.service.gov.uk/organisations/f8718311-b9a4-49d3-b1c7-7c5345a74e35)
+- GOV.UK PaaS [Space developer](https://docs.cloud.service.gov.uk/orgs_spaces_users.html#space-developer) and `Org manager`
+  access to all spaces in the [govuk_development](https://admin.cloud.service.gov.uk/organisations/f8718311-b9a4-49d3-b1c7-7c5345a74e35) and [data-gov-uk](https://admin.cloud.service.gov.uk/organisations/39c3d2c5-8809-4dcf-8cd6-a8f62923a295/users) organisations
+
+The steps above are outlined in the [GOV.UK Production template Trello card](https://trello.com/c/GIHPZi2o/382-production-access-for-2nd-line), which is normally given whilst on 2nd line.
+See the next section for more detail.
```

Comment:
I think we can lose this line.

```suggestion
```

Reply:
Thanks Chris, I've removed this change now.